### PR TITLE
Remove validator

### PR DESCRIPTION
**Pull request by mbasaglia**
It's going to be moved to the main site

### CODE REVIEW

**Review by Aidosmf — COMMENTED** [review]
should we remove:
1. tools/validate.js?
2. schema-validate.py?
3. .github/workflows/test-animation?
4. validate mentioned at "docs/editing/schema.md"?

**Comment by mbasaglia** [comment]
None of them refer to the smart Lottie validator:

> 1. tools/validate.js?

This is the script that validates the test files from the raw schema, doesn't use the validator

> 2. schema-validate.py?

This checks the schema itself

> 3. .github/workflows/test-animation?

This uses validate.js above, not the new validator 

> 4. validate mentioned at "docs/editing/schema.md"?

This mentions the schema-validate script

**Review by Aidosmf — APPROVED** [review]

**Comment by b-wils** [comment]
Changes look good, going to wait until this is on the main site to approve.

**Review by fmalita — APPROVED** [review]

**Review by b-wils — APPROVED** [review]

**Pull request merged**
